Return-Path: <nathans@wobbly.melbourne.sgi.com>
Received: from hermes.suse.de (hermes.suse.de [10.0.0.1])
	by wotan.suse.de (Postfix) with ESMTP id 5FAF0125FB
	for <olh@wotan.suse.de>; Mon,  8 Mar 2004 09:52:58 +0100 (CET)
Received: by hermes.suse.de (Postfix)
	id 590F0C73D; Mon,  8 Mar 2004 09:52:58 +0100 (CET)
Received: from Cantor.suse.de (ns.suse.de [195.135.220.2])
	(using TLSv1 with cipher EDH-RSA-DES-CBC3-SHA (168/168 bits))
	(No client certificate requested)
	by hermes.suse.de (Postfix) with ESMTP id 50E59C73C
	for <olh@suse.de>; Mon,  8 Mar 2004 09:52:58 +0100 (CET)
Received: from zok.sgi.com (mtvcafw.SGI.COM [192.48.171.6])
	by Cantor.suse.de (Postfix) with ESMTP id 456592B3624
	for <olh@suse.de>; Mon,  8 Mar 2004 09:52:56 +0100 (CET)
Received: from larry.melbourne.sgi.com (larry.melbourne.sgi.com [134.14.52.130])
	by zok.sgi.com (8.12.9/8.12.9/linux-outbound_gateway-1.1) with SMTP id i288qj02024394;
	Mon, 8 Mar 2004 00:52:46 -0800
Received: from wobbly.melbourne.sgi.com (wobbly.melbourne.sgi.com [134.14.55.135]) by larry.melbourne.sgi.com (950413.SGI.8.6.12/950213.SGI.AUTOCF) via ESMTP id TAA26215; Mon, 8 Mar 2004 19:52:42 +1100
Received: from wobbly.melbourne.sgi.com (localhost [127.0.0.1])
	by wobbly.melbourne.sgi.com (SGI-8.12.5/8.12.5) with ESMTP id i288qcFx135055;
	Mon, 8 Mar 2004 19:52:40 +1100 (EST)
Received: (from nathans@localhost)
	by wobbly.melbourne.sgi.com (SGI-8.12.5/8.12.5/Submit) id i288qYql135261;
	Mon, 8 Mar 2004 19:52:34 +1100 (EST)
Date: Mon, 8 Mar 2004 19:52:34 +1100
From: Nathan Scott <nathans@sgi.com>
To: Olaf Hering <olh@suse.de>, akpm@osdl.org, torvalds@osdl.org
Cc: linux-kernel@vger.kernel.org
Subject: Re: linux-2.5 cset 1.1654.1.2  xfs filemap_flush() unresolved
Message-ID: <20040308195234.A132331@wobbly.melbourne.sgi.com>
References: <20040307052556.4BABC468E4@mandarine.suse.de> <20040307161550.GA2812@suse.de>
Mime-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Disposition: inline
User-Agent: Mutt/1.2.5i
In-Reply-To: <20040307161550.GA2812@suse.de>; from olh@suse.de on Sun, Mar 07, 2004 at 05:15:50PM +0100
X-Spam-Checker-Version: SpamAssassin 2.63 (2004-01-11) on wotan.suse.de
X-Spam-Level: 
X-Spam-Status: No, hits=-4.9 required=5.0 tests=BAYES_00 autolearn=ham 
	version=2.63
Content-Length: 581

On Sun, Mar 07, 2004 at 05:15:50PM +0100, Olaf Hering wrote:
> 
> filemap_flush() is not an exported symbol, at least not in my tree.
> 

Thanks Olaf,

Trivial fix follows (for modular XFS builds).

cheers.

-- 
Nathan


--- ./mm/filemap.c.orig	2004-03-08 19:44:18.904099904 +1100
+++ ./mm/filemap.c	2004-03-08 19:44:45.641035272 +1100
@@ -168,6 +168,8 @@
 	return __filemap_fdatawrite(mapping, WB_SYNC_NONE);
 }
 
+EXPORT_SYMBOL(filemap_flush);
+
 /**
  * filemap_fdatawait - walk the list of locked pages of the given address
  *                     space and wait for all of them.

